website/content/docs/enterprise/index.mdx

```diff
@@ -8,22 +8,73 @@ description: >-
-- [Network Segments](/docs/enterprise/network-segments)
-- [Namespaces](/docs/enterprise/namespaces)
-- [NIA with Terraform Enterprise](/docs/nia/enterprise)
-- [Sentinel](/docs/enterprise/sentinel)
-- [OIDC Auth Method](/docs/security/acl/auth-methods/oidc)
-
-These features are part of [Consul
-Enterprise](https://www.hashicorp.com/consul).
+[Consul Enterprise](https://www.hashicorp.com/products/consul/)
+features address the organizational complexities of collaboration, operations, scale, and governance
+and are [available in several forms of Consul Enterprise](#consul-enterprise-feature-availability).
+If you have purchased Consul Enterprise, refer to
+[how to gain access to enterprise features](#gain-access-to-enterprise-features).
+
+Enterprise features include:
+
+#### Multi-Tenancy
+- [Admin Partitions](/docs/enterprise/admin-partitions): define administrative boundaries between tenants within a single Consul datacenter
+- [Namespaces](/docs/enterprise/namespaces): define resource boundaries within a single admin partition for further organizational flexibility
+
+#### Resiliency
+- [Automated Backups](/docs/enterprise/backups): configure the automatic backup of Consul state
+- [Redundancy Zones](/docs/enterprise/redundancy): deploy backup voting Consul servers to efficiently improve Consul fault tolerance
+
+#### Scalability
+- [Read Replicas](/docs/enterprise/read-scale): deploy non-voting Consul servers to enhance the scalability of read requests
+
+#### Operational Simplification
+- [Automated Upgrades](/docs/enterprise/upgrades): ease upgrades by automating the transition from existing to newly deployed Consul servers
+- [Consul-Terraform-Sync Enterprise](/docs/nia/enterprise): leverage the enhanced network infrastructure automation capabilities of the enterprise version of Consul-Terraform-Sync
+
+#### Complex Network Topology Support
+- [Network Areas](/docs/enterprise/federation): support complex network topologies between federated Consul datacenters with pairwise federation rather than full mesh federation
+- [Network Segments](/docs/enterprise/network-segments): support complex network topologies within a Consul datacenter by enforcing boundaries in Consul client gossip traffic
+
+#### Governance
+- [OIDC Auth Method](/docs/security/acl/auth-methods/oidc): manage user access to Consul through an OIDC identity provider instead of Consul ACL tokens directly
+- [Audit Logging](/docs/enterprise/audit-logging): understand Consul access and usage patterns by reviewing access to the Consul HTTP API 
+- [Sentinel for KV](/docs/enterprise/sentinel): policy-as-code framework for defining advanced key-value storage access control policies
+
+# Gain Access to Enterprise Features
+
+### Self-Managed Consul
+
+To gain access to Consul Enterprise features in a self-managed installation,
+[apply a purchased license](/docs/enterprise/license/overview)
+to the Consul Enterprise binary that grants access to the desired features.
+
```

Comment:
```suggestion
To gain access to Consul Enterprise features in a self-managed installation,
[apply a purchased license](/docs/enterprise/license/overview)
to the Consul Enterprise binary that grants access to the desired features.
You can also try out Consul Enterprise through a 30 day trial license. Visit [https://consul.io/trial](https://consul.io/tria) for a trial license. 

```
I think this will help guide users towards the trial page :)

Reply:
I'm also going to add a similar call-out to the HCP section.